src/Board.js

```diff
@@ -1,56 +1,57 @@
 import { useState } from 'react'
 import Square from './Square.js'
 
-const board = [...Array(10).keys()]
-const filled = 'red'
-const unfilled = 'grey'
+const board = [...Array(34).keys()]
```

Comment:
It seems like you might just be using the `length` of this now, right? So it can probably just be a number

Reply:
Nope. Still doing the same thing as before to create the `grid` with it where I'm mapping on it twice to get row and column numbers.